kore/app/exec/Main.hs

```diff
@@ -620,59 +635,87 @@ mainDispatch = warnProductivity . mainDispatchWorker
                 then koreBmc localOptions proveOptions
                 else koreProve localOptions proveOptions
         | Just searchOptions <- koreSearchOptions =
-            koreSearch localOptions searchOptions
+            koreSearch exeLastModifiedTime localOptions searchOptions
+        | True <- serialize =
+            koreSerialize localOptions
```

Comment:
So if you give it the serialize flag it will only serialize the definition.

Reply:
Well, a lot of the logic needed to do this was part of the Kore.Exec and kore/app/exec/Main files, so it seemed more appropriate to put it here than to do a huge refactoring of where a whole bunch of code lived. As it was, I already had to move around a fair bit.